Simplified/Settings/NYPLSettings+OE.swift

```diff
@@ -31,6 +31,7 @@ extension NYPLSettings {
     }
     set(newAccountsList) {
       UserDefaults.standard.set(newAccountsList, forKey: NYPLSettings.settingsLibraryAccountsKey)
+        // synchronize() is deprecated
```

Comment:
I believe only since iOS 12 though? it's funny the docs still don't label it as such. We could add a "if iOS 12+" kind of check though.